### DISCUSSION TITLE
Does fingerprintjs detect undetected-chromedriver library for Selenium?

### DISCUSSION THREAD

**Discussion by zapmaxvevo** [fingerprintjs/BotD#168] [Q&A] [answered]
Hello, sorry if this is not the way to ask this things but i was exploring differents approachs to bot detection in my website and i found this library. 

While i was doing some simple test i found that the demo page doesn't detected my selenium using the undetected-chromedriver.

I was wondering, does it normally detect this behaviour?

This is the code i was using in my selenium using Google Chrome 125.0.6422.141 Selenium 4.21.0 and undetected-chromedriver 3.5.5

`from selenium.webdriver.chrome.options import Options
from selenium.webdriver.chrome.service import Service
from webdriver_manager.chrome import ChromeDriverManager
import undetected_chromedriver as uc

options = Options()
options.add_argument('--auto-open-devtools-for-tabs')
options.add_argument('--disable-dev-shm-usage')
driver = uc.Chrome(service=Service(ChromeDriverManager().install()), options=options)

driver.get("https://fingerprintjs.github.io/BotD/main/") `

Any help, including how to word this better will be appreciated!

**Comment by xnerhu** [comment, marked as answer]
Hello @Maxi-Panza 

BotD OSS doesn't support sophisticated bots like undetected-chromedriver, however our BotD Pro does.
We don't plan to support UC in BotD OSS.

https://fingerprint.com/products/bot-detection/

If you have more questions, let me know.

**Reply by zapmaxvevo** [reply]
Thank you very much for your response and aclaration!
